addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -0,0 +1,429 @@
+<?xml version="1.0" ?>
+<odoo><data>
+
+<!-- Channels home template -->
+<template id='courses_home' name="Odoo Courses Homepage">
+    <t t-call="website.layout">
+        <t t-call="website_slides.courses_header">
+            <t t-set="header_title" t-value="'Reach new heights'"/>
+            <t t-set="header_subtitle" t-value="'Start your online course today !'"/>
+            <t t-set="header_text" t-value="'Skill up and have an impact! Your business career starts here. Time to start a course.'"/>
+            <t t-set="header_cta_url" t-value="'slides'"/>
+            <t t-set="header_cta_title" t-value="'Pick a course'"/>
+        </t>
+        <div class="wrap">
+            <div class="container mt16 o_wslides_home_nav">
+                <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                    <button class="navbar-toggler" type="button"
+                        data-toggle="collapse" data-target="#navbarSlidesHomepage"
+                        aria-controls="navbarSlidesHomepage" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarSlidesHomepage">
+                        <a class="navbar-brand" href="#"></a>
+                        <ul class="navbar-nav mr-auto">
+                            <li class="nav-item">
+                                <h4><a class="nav-link" href="/slides/all"><i class="fa fa-rocket"/> All courses</a></h4>
+                            </li>
+                            <li class="nav-item ml16">
+                                <h4><a class="nav-link" href="/slides/all"><i class="fa fa-certificate"/> Certifications</a></h4>
```

Comment:
I think we use fa-trophy everywhere else for certifications.

Reply:
Added to my cleaning pass.